nix/tests/container-test/default.nix

```diff
@@ -7,29 +7,29 @@ let
     # Found via https://hub.docker.com/_/ubuntu/ under "How is the rootfs build?"
     # Jammy
     "ubuntu-v22_04" = {
-      tarball = import <nix/fetchurl.nix> {
-        url = "https://launchpad.net/~cloud-images-release-managers/+livefs/ubuntu/jammy/ubuntu-oci/+build/408115/+files/livecd.ubuntu-oci.rootfs.tar.gz";
-        hash = "sha256-BirwSM4c+ZV1upU0yV3qa+BW9AvpBUxvZuPTeI9mA8M=";
+      tarball = builtins.fetchurl {
+        url = "http://cdimage.ubuntu.com/ubuntu-base/releases/22.04/release/ubuntu-base-22.04-base-amd64.tar.gz";
+        sha256 = "sha256:01sbpjb32x1z1yr9q78zrk0a6kfw5c4fxw1jqmm23g8ixryffvyz";
       };
       tester = ./default/Dockerfile;
       system = "x86_64-linux";
     };
 
     # focal
     "ubuntu-v20_04" = {
-      tarball = import <nix/fetchurl.nix> {
-        url = "https://launchpad.net/~cloud-images-release-managers/+livefs/ubuntu/focal/ubuntu-oci/+build/408120/+files/livecd.ubuntu-oci.rootfs.tar.gz";
-        hash = "sha256-iTJR+DeC5lT4PMqT/xFAFwmlC/qvslDFccDrVFLt/a8=";
+      tarball = builtins.fetchurl {
+        url = "http://cdimage.ubuntu.com/ubuntu-base/releases/20.04/release/ubuntu-base-20.04.1-base-amd64.tar.gz";
+        sha256 = "sha256:0ryn38csmx41a415g9b3wk30csaxxlkgkdij9v4754pk877wpxlp";
```

Comment:
I don't usually see the `sha256:` prefix.

Reply:
It's what nix outputs....
![image](https://user-images.githubusercontent.com/130903/218593696-34a1f241-072e-4553-9e2e-68abaab88c1e.png)

I'll yank it though.